app/components/shared/continue_or_cancel_button_group.html.erb

```diff
@@ -7,7 +7,7 @@
       form: form_id,
       data_attributes: {
         "testid": "submit-button"
-      }
+      },
```

Comment:
We tend to rebase any lint failures into their relevant commits - I find [`git absorb`](https://github.com/tummychow/git-absorb) really useful for this